src/refactor_nrepl/externs.clj

```diff
@@ -0,0 +1,28 @@
+(ns refactor-nrepl.externs
+  (:import java.net.URL
+           org.projectodd.shimdandy.ClojureRuntimeShim
+           java.net.URLClassLoader
+           java.util.ArrayList))
+
+(def rt (atom nil))
+
+(defn create-runtime []
+  (let [urls [(URL. "file:libs/shimdandy-impl-1.1.0.jar")
+              (URL. "file:libs/clojure-1.6.0.jar")
+              (URL. "file:libs/pomegranate-0.3.1-SNAPSHOT.jar")]
+        cl (URLClassLoader. (into-array URL urls))]
+    (reset! rt (ClojureRuntimeShim/newRuntime cl "refactor-nrepl"))))
+
+(defn add-dependencies [dep repos]
+  (when-not @rt
+    (create-runtime))
+  (.require @rt (into-array String ["cemerick.pomegranate"]))
```

Comment:
I'm not sure what's going on there. Can you give me a quick HOWTO on how I can recreate it here?